src/db/mapper.py

```diff
@@ -0,0 +1,15 @@
+import combatants
+from db import schema
+
+
+class FromTableMapper:
+
+    def map_combatant(self, obj: schema.Combatant) -> combatants.Combatant:
```

Comment:
Please name the combatant classes different way so that it would be clear what this function is doing.

Reply:
`Combatant(**asdict(obj))`